src/features/Classes/ClassesFilters/index.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { Col, Form } from '@edx/paragon';
+import { Select, Button } from 'react-paragon-topaz';
+import { logError } from '@edx/frontend-platform/logging';
+
+import { initialPage } from 'features/constants';
+import { fetchCoursesData } from 'features/Courses/data/thunks';
+import { fetchClassesData } from 'features/Classes/data/thunks';
+import { fetchInstructorsData } from 'features/Instructors/data/thunks';
+import { updateFilters, updateCurrentPage } from 'features/Licenses/data/slice';
+
+const notAssignedOption = {
+  label: 'Not assigned',
+  value: 'null',
+};
+
+const ClassesFilters = ({ resetPagination }) => {
+  const dispatch = useDispatch();
+  const institution = useSelector((state) => state.main.selectedInstitution);
+  const courses = useSelector((state) => state.courses.table.data);
```

Comment:
Data for the selectors is different from the tables, the query limit is false, instead of pages we get all data

Reply:
What do you mean about @AuraAlba 

> Data for the selectors is different from the tables

Reply:
there is a query param in this service called 'limit', for the tables we used it as true because give pagination but for the selector in filters should be false, so we have to save this data in other part of the state, please see students filters to see how is the response

Reply:
Got the same error here, for the courses and instructor selectors you shouldn't use the information get on table because the services has limit true, and must be false to get all option. Please store that information in a different selector